mmv1/third_party/terraform/fwtransport/framework_config.go.erb

```diff
@@ -31,25 +31,26 @@ import (
 )
 
 type FrameworkProviderConfig struct {
-	BillingProject             types.String
-	Client                     *http.Client
-	Context                    context.Context
-	gRPCLoggingOptions         []option.ClientOption
-	PollInterval               time.Duration
-	Project                    types.String
-	Region                     types.String
-	Zone                       types.String
-	RequestBatcherIam          *transport_tpg.RequestBatcher
-	RequestBatcherServiceUsage *transport_tpg.RequestBatcher
-	Scopes                     types.List
-	TokenSource                oauth2.TokenSource
-	UserAgent                  string
-	UserProjectOverride        types.Bool
-
-	// paths for client setup
-	<% products.each do |product| -%>
-	<%= product[:definitions].name -%>BasePath string
-	<% end -%>
+    BillingProject types.String
+    Client *http.Client
+    Context context.Context
+    gRPCLoggingOptions []option.ClientOption
+    PollInterval time.Duration
+    Project types.String
+    Region types.String
+    Zone types.String
+    RequestBatcherIam *transport_tpg.RequestBatcher
+    RequestBatcherServiceUsage *transport_tpg.RequestBatcher
+    Scopes []string
+    TokenSource oauth2.TokenSource
+	  UniverseDomain types.String
```

Comment:
nit: spacing (these should all be tabs)

Reply:
I will handle spacing problems at the end.